src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -84,6 +91,8 @@ public function setInputStream($stream)
     /**
      * Returns the helper's input stream.
      *
+     * @deprecated Will be removed in 4.0, use StreamableInputInterface::getStream() instead
```

Comment:
Not sure that we are talking about the same thing, my description is surely not clear.
 I introduced a new `StreamableInputInterface` implemented by `Symfony\Component\Console\Input\Input` containing two methods `getStream()` and  `setStream()`.
These methods would replace the old `QuestionHelper::getInputStream()` and `QuestionHelper::setInputStream()`, that's why I depreciate them here.
The problem is that `Application::configureIO` get the stream from `QuestionHelper::getInputStream()` before, so we need to keep it intact before totally remove it later. But, this call causes deprecations in all ApplicationTest tests that call `Application::run()` because of the call of the deprecated `QuestionHelper::getInputStream()`.